core/domain/exp_jobs_one_off.py

```diff
@@ -506,3 +506,45 @@ def map(exp_summary_model):
     def reduce(key, values):
         """Implements the reduce function for this job."""
         yield (key, len(values))
+
+
+class RegenerateMissingExpCommitLogModels(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
Confirming that this doesn't occur anymore since a commit to exp model always creates commit log model: https://github.com/oppia/oppia/blob/develop/core/storage/exploration/gae_models.py#L180. We should still work on #10808 for eliminating this completely even by accident.